content/en-us/docs/khala-mining/1-1-hardware-requirements.md

```diff
@@ -10,52 +10,301 @@ menu:
 
 A system can potentially mine Phala if it meets these general requirements:
 
-![](/images/docs/poc3/1-3.1.png)
+![](/images/docs/khala-mining/mining_requirements_table.png)
 
-It will also need a motherboard and BIOS which support using Intel SGX to run a Trusted Execution Environment (TEE).
+### Requirements Checklist
+
+- [x] An [Intel® SGX](https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html)  compatible processor. 
+   - [Lookup your CPU](#1-lookup-your-processor) and see if it is compatible.
+   - [How to find new Intel® SGX processors](#2-confirm-the-cpu-supports-intel-sgx) if mine is not compatible.
+- [x] A motherboard and BIOS that supports [Intel® SGX](https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html) to run the [Trusted Execution Environment (TEE)](https://murdoch.is/talks/rhul14tee.pdf) is required.
+   - [Check BIOS compatability](#check-your-bios).
+- [x] A Solid-state drive (SSD) storage device
+   - Storing your blockchain data on a mechanical HDD will result in slow syncronosation speeds. At a minimum a 512GB SSD drive is recommended. 
+      - Refer to [issue #554](https://github.com/Phala-Network/phala-blockchain/issues/554) for more info.
+- [x] Have a supported version of Ubuntu (18.04, 20.04, 21.04) installed and booted from it.
+   - See [Suported OS requirements](#supported-operating-systems). 
+- [x] You require a minimum amount of 3 PHA to mine
```

Comment:
This sentence can be confusing, cuz miners may think 3 PHA is enough for mining.

Reply:
removed sentence & restructured